gc/base/standard/Scavenger.cpp

```diff
@@ -3343,25 +3331,10 @@ MM_Scavenger::masterThreadGarbageCollect(MM_EnvironmentBase *envBase, MM_Allocat
 	}
 
 #if defined(OMR_GC_CONCURRENT_SCAVENGER)
-	if (_extensions->concurrentScavenger) {
-		if (concurrent_state_idle == _concurrentState) {
-			_concurrentState = concurrent_state_init;
-			_forceConcurrentTermination = false;
-		}
-		scavengeIncremental(env, 3);
-		if (concurrent_state_scan == _concurrentState) {
-			_activeSubSpace->flip(env, MM_MemorySubSpaceSemiSpace::set_evacuate);
-			_activeSubSpace->flip(env, MM_MemorySubSpaceSemiSpace::set_allocate);
-
-		}
-	}
-	else
+	scavengeIncremental(env, 3);
```

Comment:
3ms default increment duration, if/when incremental mode is implemented. It's just a remnant of other Garbage Collectos that rely on MasterGCThread infrastructures, and it's ignored for now. That and some other code within scavengeIncremental method is subject to removal or will evolve in something more meaningful, time permitting.